modules/library/coverage/src/main/java/org/geotools/coverage/processing/operation/Resampler2D.java

```diff
@@ -243,13 +300,17 @@ public static GridCoverage2D reproject(
      * @throws TransformException if a transformation failed.
      */
     public static GridCoverage2D reproject(
-            GridCoverage2D sourceCoverage,
-            CoordinateReferenceSystem targetCRS,
-            GridGeometry2D targetGG,
-            Interpolation interpolation,
+            final GridCoverage2D sourceCoverage,
+            final CoordinateReferenceSystem targetCRS,
+            final GridGeometry2D targetGG,
+            final Interpolation interpolation,
             final Hints hints,
             final double[] backgroundValues)
             throws FactoryException, TransformException {
+
+        ResampleState state;
+        GridCoverage2D targetCoverage;
+
         ////////////////////////////////////////////////////////////////////////////////////////
         ////                                                                                ////
         //// =======>>  STEP 1: Extracts needed informations from the parameters   <<====== ////
```

Comment:
```suggestion
        ////  STEP 1: Extracts needed information from the parameters  ////
```

This comment has become a summary of what the method does, not indicating the step any more